mmv1/templates/terraform/examples/osconfigv2_policy_orchestrator_for_folder_basic.tf.tmpl

```diff
@@ -0,0 +1,104 @@
+resource "google_folder" "my_folder" {
+    display_name        = "{{index $.Vars "folder"}}"
+    parent              = "organizations/{{index $.TestEnvVars "org_id"}}"
+    deletion_protection = false
+}
+
+resource "google_folder_service_identity" "osconfig_sa" {
+  folder  = google_folder.my_folder.folder_id
+  service = "osconfig.googleapis.com"
+}
+
+resource "google_folder_service_identity" "ripple_sa" {
+  folder  = google_folder.my_folder.folder_id
+  service = "progressiverollout.googleapis.com"
+}
+
+resource "time_sleep" "wait_30_sec" {
+    depends_on = [
+        google_folder_service_identity.osconfig_sa,
+        google_folder_service_identity.ripple_sa,
+    ]
+    create_duration = "30s"
+}
+
+resource "google_folder_iam_member" "iam_osconfig_service_agent" {
```

Comment:
so, the problem here is not specifically google_folder_iam_member - it's that the order of the `google_folder_iam_member` resources is non-deterministic. You can fix this by specifying an explicit order using [`depends_on`](https://developer.hashicorp.com/terraform/language/meta-arguments/depends_on) and then you shouldn't need to skip VCR!